Dockerfile

```diff
@@ -1,5 +1,9 @@
 FROM mas.ops.maap-project.org/root/jupyter-image/vanilla:develop
 
+COPY requirements.txt requirements.txt
```

Comment:
ipykernel and papermill are already installed in the image, and use the same versions as specifiied in requirements.txt 

pytest is the only library that's not already installed in the image